deploy/s3/deployment.yaml

```diff
@@ -56,7 +56,12 @@ metadata:
   namespace: REPLACE_NAMESPACE
 spec:
   ports:
-    - port: 9000
+    - name: object-store
+      port: 9000
       protocol: TCP
+      targetPort: 9000
   selector:
     app: object-store
+  type: LoadBalancer
+status:
```

Comment:
I don't think you need this

Reply:
I just copied from the same way we deploy our other services.
If you think it's not required we could remove it from all of them on a separate PR